src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -112,6 +112,11 @@ class GemUtils {
 
                 systemProperties 'file.encoding' : 'utf-8'
             }
+            project.javaexec {
```

Comment:
> Yes, we can. The install process have proven itself, so we can towards some tweaks. I do not want to lose the ability to run out of process, as the isolation it provides means is great. However, I think it would relatively easy to have an oprion to run in-process. The experimental work I have done for embedding Rake can refactored so that appropriate code ends up in base.

@ysb33r this isn't necessarily what I meant by "avoiding spinning up another JVM." Right now we already spin up one JVM for JRuby to execute `gem install` (effectively), this change means that the `jrubyPrepareGems` task will spin up *two* JVMs to accomplish its work instead of one.

I'd be open to changing this behavior to remove any sub-JVM processes, but I'm not sure we can/should accomplish that within the scope of this change from @mkristian